src/components/ApeToggleNew/ApeToggleNew.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react';
+
+import * as ToggleGroup from '@radix-ui/react-toggle-group';
+import uniqueId from 'lodash/uniqueId';
+import { styled } from 'stitches.config';
+
+import { ApeInfoTooltip } from 'components';
+import { Box } from 'ui';
+
+const StyledToggleGroupRoot = styled(ToggleGroup.Root, {
+  fontWeight: 300,
+  variants: {
+    locked: {
+      true: {
+        opacity: 0.6,
+      },
+    },
+  },
+});
+const StyledToggleGroupItem = styled(ToggleGroup.Item, {
+  minWidth: 102,
+  borderRadius: '0 8px 8px 0',
+  '&:not(:last-child)': {
+    borderRight: '1px solid white',
+    borderRadius: '8px 0 0 8px',
+  },
+  color: '$white',
+  backgroundColor: '#A8B1B4',
+
+  textTransform: 'none',
+  textAlign: 'center',
+  fontSize: 19,
+  lineHeight: 1.26,
+  padding: '12px 24px',
+  variants: {
+    active: {
+      true: {
+        color: '$white',
+        background: '$lightBlue',
```

Comment:
as you'll see if you look at the main branch, we've changed the color names. Please use `$primary` as the background for active toggles and `$surface` for inactive ones. Just set `opacity: 0.8` on hover instead of changing colors.